tests/feature_tests/unary_arithmetic.c

```diff
@@ -0,0 +1,14 @@
+int main() {
```

Comment:
Could you add:
- a test negating an `unsigned char`? this will confirm the type promotion is working as we expect
- an error test for when the argument to unary + or - is not of arithmetic type

The latter can be done in a new file, perhaps called error_unary.c.